Makefile

```diff
@@ -53,7 +54,7 @@ deploy:
 func-test: deploy
 	@echo "Running functional test suite"
 	go clean -testcache
-	go test -timeout 20m -v ./functests/...
+	go test -timeout 20m -v ./functests/... || (oc -n openshift-updateservice adm inspect --dest-dir="$(ARTIFACT_DIR)/inspect" namespace/openshift-updateservice customresourcedefinition/updateservices.updateservice.operator.openshift.io updateservice/example; false)
```

Comment:
Could you add a comment on why the clause is here?

Reply:
It's one of two lines touched by f37667556f6f4e20b7833f6252482be6830a731e which explains the addition.  I'm fine inlining that commit message in a Makefile comment for folks who prefer to not use `blame`, if folks want.  I'm also fine dropping the commit from the pull once I get CI greened up.